src/ASM/ASMs1D.h

```diff
@@ -466,8 +466,8 @@ class ASMs1D : public ASMstruct, public ASMunstruct, public ASM1D
   virtual void getElmConnectivities(IntMat& neighs) const;
 
 protected:
-  Go::SplineCurve* curv; //!< Pointer to the actual spline curve object
-  Go::SplineCurve* proj; //!< Pointer to spline curve for projection basis
+  std::shared_ptr<Go::SplineCurve> curv; //!< Pointer to the actual spline curve object
+  std::shared_ptr<Go::SplineCurve> proj; //!< Pointer to spline curve for projection basis
```

Comment:
Then remove "Pointer to" in the doxy to keep it within 80 columns.

Applies elsewhere in similar declarations as well.

Reply:
ok.